src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -270,6 +270,28 @@ function ReportActionItemMessageEdit(props) {
         [deleteDraft, isKeyboardShown, isSmallScreenWidth, publishDraft],
     );
 
+    /**
+     * Focus the composer text input
+     */
+    const focus = React.useCallback(() => {
+        // There could be other animations running while we trigger manual focus.
+        // This prevents focus from making those animations janky.
+        InteractionManager.runAfterInteractions(() => {
+            if (!textInputRef.current) {
+                return;
+            }
+
+            // Keyboard is not opened after Emoji Picker is closed
+            // SetTimeout is used as a workaround
+            // https://github.com/react-native-modal/react-native-modal/issues/114
+            // We carefully choose a delay. 100ms is found enough for keyboard to open.
+            setTimeout(() => {
+                setIsFocused(true);
+                textInputRef.current.focus();
+            }, 100);
+        });
```

Comment:
Can we make utils function for this? just to avoid duplicated code

Reply:
mSafari not working because of this known issue - #10414 (keyboard doesn't show though highlighted)
Same happens on main composer

Reply:
The original issue is for iOS native

Reply:
Hey, 
This PR matches the composer's behavior in editing mode with the composer for new messages. 
It fixes displaying the keyboard (focusing) on native and mWeb Chrome. It doesn't address the mWeb Safari keyboard issues.